pkg/progress/action.go

```diff
@@ -144,6 +144,26 @@ func completedOrFailed(aIDX int, actionSet *crv1alpha1.ActionSet, phaseName stri
 	return false
 }
 
+func isPhaseProgressDifferent(a, b crv1alpha1.PhaseProgress) bool {
```

Comment:
just one small nit please feel free to ignore it. because we have two PhaseProgresses here, should we rename this to 
`arePhaseProgressesDifferent`?

Reply:
I thought about this yesterday but my conclusion was - probably it's not so important. But since you have the same thoughts, probably that makes sense. Renamed.